packages/angular_devkit/build_angular/package.json

```diff
@@ -11,52 +11,52 @@
     "@angular-devkit/build-webpack": "0.0.0",
     "@angular-devkit/core": "0.0.0",
     "@ngtools/webpack": "0.0.0",
-    "ajv": "~6.4.0",
-    "autoprefixer": "^8.4.1",
-    "circular-dependency-plugin": "^5.0.2",
-    "clean-css": "^4.1.11",
-    "copy-webpack-plugin": "^4.5.2",
-    "file-loader": "^1.1.11",
-    "glob": "^7.0.3",
-    "html-webpack-plugin": "^3.0.6",
-    "istanbul": "^0.4.5",
-    "istanbul-instrumenter-loader": "^3.0.1",
-    "karma-source-map-support": "^1.2.0",
-    "less": "^3.7.1",
-    "less-loader": "^4.1.0",
-    "license-webpack-plugin": "^1.3.1",
-    "loader-utils": "^1.1.0",
-    "mini-css-extract-plugin": "~0.4.0",
-    "minimatch": "^3.0.4",
-    "parse5": "^4.0.0",
-    "opn": "^5.1.0",
-    "portfinder": "^1.0.13",
-    "postcss": "^6.0.22",
-    "postcss-import": "^11.1.0",
-    "postcss-loader": "^2.1.5",
-    "postcss-url": "^7.3.2",
-    "raw-loader": "^0.5.1",
-    "rxjs": "~6.2.0",
-    "sass-loader": "^7.1.0",
-    "semver": "^5.5.0",
-    "source-map-support": "^0.5.0",
-    "source-map-loader": "^0.2.3",
-    "stats-webpack-plugin": "^0.6.2",
-    "style-loader": "^0.21.0",
-    "stylus": "^0.54.5",
-    "stylus-loader": "^3.0.2",
-    "tree-kill": "^1.2.0",
-    "terser-webpack-plugin": "^1.0.2",
-    "url-loader": "^1.0.1",
-    "webpack": "^4.15.1",
-    "webpack-dev-middleware": "^3.1.3",
-    "webpack-dev-server": "^3.1.4",
-    "webpack-merge": "^4.1.2",
-    "webpack-sources": "^1.1.0",
-    "webpack-subresource-integrity": "^1.1.0-rc.4"
+    "ajv": "6.5.3",
+    "autoprefixer": "9.1.5",
+    "circular-dependency-plugin": "5.0.2",
+    "clean-css": "4.2.1",
+    "copy-webpack-plugin": "4.5.2",
+    "file-loader": "2.0.0",
+    "glob": "7.1.3",
+    "html-webpack-plugin": "3.2.0",
+    "istanbul": "0.4.5",
+    "istanbul-instrumenter-loader": "3.0.1",
+    "karma-source-map-support": "1.3.0",
+    "less": "3.8.1",
+    "less-loader": "4.1.0",
+    "license-webpack-plugin": "1.4.0",
+    "loader-utils": "1.1.0",
+    "mini-css-extract-plugin": "0.4.2",
+    "minimatch": "3.0.4",
+    "parse5": "4.0.0",
+    "opn": "5.3.0",
+    "portfinder": "1.0.17",
+    "postcss": "7.0.2",
+    "postcss-import": "12.0.0",
```

Comment:
Avoid major version bumps for now.  We should analyze them first to see if there are any relevant breaking changes and update them in a separate PR.

Reply:
Updated